### DISCUSSION TITLE
Put desklet on the loginscreen

### DISCUSSION THREAD

**Discussion by mirohh-commits** [linuxmint/discussions#589] [Ideas]
I want put a desklet on the loginscreen before a user is authenticated. in this case the hostcheck from schorschii. Is that possible?
So everyone can see if all servers are online, without login. i have LM 22 with cinnamon.
tks miro

**Comment by JosephMcc** [comment]
No, it is not. Desklets only work under Cinnamon. On the login screen you have not yet booted into a desktop environment.
